conda/misc.py

```diff
@@ -341,3 +344,18 @@ def clone_env(prefix1, prefix2, verbose=True, quiet=False, index_args=None):
         index_args=index_args,
     )
     return actions, untracked_files
+
+
+def _get_best_prec_match(precs):
+    assert precs
+    for chn in context.channels:
+        channel_matcher = ChannelMatch(chn)
+        prec_matches = tuple(
+            prec for prec in precs if channel_matcher.match(prec.channel.name)
+        )
+        if prec_matches:
+            break
+    else:
+        prec_matches = precs
+    log.warn("Multiple packages found:%s", dashlist(prec_matches))
```

Comment:
```suggestion
    log.warn("Multiple packages found: %s", dashlist(prec_matches))
```

Reply:
FYI the space is usually skipped since `dashlist` starts with a newline character